examples/quantization_2of4_sparse_w4a16/llama7b_sparse_w4a16.py

```diff
@@ -16,6 +13,9 @@
 # uses LLM Compressor's built-in preprocessing for ultra chat
 dataset = "ultrachat-200k"
 
+# Select the recipe for 2 of 4 sparsity and 4-bit activation quantization
+recipe = "2of4_w4a16_recipe.yaml"
```

Comment:
do we need an absolute path here? Something like this would allow it to run from any working directory
```suggestion
from pathlib import Path
recipe = str(Path(__file__).parent / "2of4_w4a16_recipe.yaml")
```